docs/source/procedures/datadog/index.rst

```diff
@@ -0,0 +1,53 @@
+=====================================
+Scylla Monitoring Datadog Integration
+=====================================
+
+.. toctree::
+   :maxdepth: 2
+   :hidden:
+
+   Alerting <alerting>
+
+
+The safest way to use Datadog with Scylla is to load the metrics from a Prometheus server and not from Scylla directly.
+The integration consists of:
+
+1. Installing and configuring the Datadog client.
+2. Loading the dashboard.
+3. Optionally load Monitor (Alerts).
+
+Scylla Cloud users, use and update the proper configuration file.
+
+Install And configure the Datadog agent
+======================================
+
+Start by following `Installation <https://docs.datadoghq.com/agent/>`_ guide.
+
+Once the Datadog agent is working download the configuration file and place it under /etc/datadog-agent/conf.d/prometheus.d/conf.yaml
+
+Scylla Cloud Users
+^^^^^^^^^^^^^^^^^^
+Scylla Cloud users, download the configuration file :download:`conf.yaml <cloud-conf.yaml>` move it to: /etc/datadog-agent/conf.d/prometheus.d/conf.yaml
+
+
+Edit the file. You must replace the cluster id (CLUSTER_ID) and the token (TOKEN).
+
+Other Scylla Users
+^^^^^^^^^^^^^^^^^^
+Other Scylla users, download the configuration file :download:`conf.yaml <conf.yaml>` and replace the ip address of the promemtheus server.
+
+
+Post configuration
+^^^^^^^^^^^^^^^^^^
+Restart the agent based on your installation. Scylla metrics should be visible in Datadog.
+
+
+Upload the Dashbaord
+====================
+Download the dashboard file :download:`dashboard.json <dashboard.json>`.
```

Comment:
Is it the same dashboard of Scylla Cloud and Scylla? not clear

Reply:
It is the same dashboard, the only difference is the configuration file